Formula/lib/libmapper.rb

```diff
@@ -0,0 +1,45 @@
+class Libmapper < Formula
+  desc "Distributed system for media control mapping"
+  homepage "http://www.libmapper.org"
+  url "https://github.com/libmapper/libmapper/releases/download/2.4.2/libmapper-2.4.2.tar.gz"
+  sha256 "130d0761cd6006c2f7458771ae0484d53d601d85dba9315cdffd4da4c30bf86b"
+  license "LGPL-2.1-or-later"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => [:build, :test]
+  depends_on "liblo"
+
+  uses_from_macos "zlib"
+
+  def install
+    args = %W[
+      --disable-debug
+      --disable-dependency-tracking
+      --prefix=#{prefix}
+    ]
+
+    if build.head?
+      system "./autogen.sh", *args
+    else
+      system "./configure", *args
+    end
```

Comment:
Since there is no HEAD defined
```suggestion
    system "./configure", *std_configure_args
```